### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by ChaseC99**
Created a section for student testimonials! We'll need to check in with these students first to make sure they are ok with us using their quote. We can also give them the option to remain anonymous.

| Light | Dark | Mobile |  
| --- | --- | --- |
| image | image | image |

Also fixed a spacing issue with FeatureList on mobile

### CODE REVIEW

**Comment by ChaseC99** [comment]
I've messaged all 9 of them. So far we have approval from:
- [ ] Yaseen
- [ ] Audrey
- [ ] Ye
- [ ] Anthony
- [ ] Michelle
- [x] Wesley
- [ ] Tanush
- [ ] Solvin
- [ ] Esteban

**Review by ramanxg — APPROVED** [review]
Awesome, I'll approve for now, but we can merge once everyone gives the OK for their testimonial.

**Pull request merged**
